packager/media/formats/mp2t/ac3_header.cc

```diff
@@ -103,14 +103,28 @@ size_t Ac3Header::GetHeaderSize() const {
   return 0;
 }
 
-size_t Ac3Header::GetFrameSize() const {
+/// Calculate the size of the frame from the sample rate code and the 
+/// frame size code.
+/// @return the size of the frame (header + payload).
```

Comment:
nit: Use normal comment style for internal functions, e.g.

// Calculate the size of the frame from the sample rate code and the 
// frame size code.

Reply:
Ah! I wondered. Should've checked the style guide.